rackcity/models/asset.py

```diff
@@ -17,12 +17,12 @@ def get_next_available_asset_number():
             return asset_number
 
 
-def get_assets_for_cp(change_plan=None):
+def get_assets_for_cp(change_plan, show_decommissioned=False):
```

Comment:
made change_plan a required param, and also added optional param to return decomissioned  assets on cp